e2e_tests/test_search.py

```diff
@@ -149,3 +176,87 @@ def test_click_on_clear_all_as_standard_user_redirects_to_browse_transferring_bo
         standard_user_page.get_by_role("link", name="Clear all").click()
 
         expect(standard_user_page).to_have_url(url)
+
+
+class TestOpenSearchResults:
+    @property
+    def search_route_url(self):
+        return "/search_results_summary"
+
+    @property
+    def browse_transferring_body_route_url(self):
+        return "/search/transferring_body"
+
+    @property
+    def transferring_body_id(self):
+        return "c3e3fd83-4d52-4638-a085-1f4e4e4dfa50"
+
+    def test_test_search(self, aau_user_page: Page):
+        """
+        Given a valid search query is entered on the search page
+        When the query is submitted
+        Then the search_results_summary page should display relevant results,
+        ensuring the results are fetched from OpenSearch and shown with correct metadata formatting
+        And when a transferring body is clicked on
+        Then the search_results/transferring_body/<body_id> page should display the filtered results correctly
+        """
+        url = f"{self.search_route_url}?query=a&search_area=everywhere"
```

Comment:
is the purose of this test any different to `test_search_returns_results_summary` in the above test class?